http/config.json5

```diff
@@ -79,7 +79,7 @@ with open(sys.argv[1], 'w') as html: \n\
     build: {
         stages: {
             base: {
-                'from': 'ubuntu:20.04',
+                'from': 'ubuntu:jammy',
```

Comment:
Can we specify this as 22.04 instead of jammy?

Reply:
I'm fine both with switching to 22.04 as well as updating the version throughout the rest of the FAW.